### DISCUSSION TITLE
obs won't open after installation

### DISCUSSION THREAD

**Discussion by arize99** [royshil/obs-backgroundremoval#599] [General]
First I used the zip package and added the files in the folders, OBS didn't launch after that, even after restarting the PC. But OBS launched after I removed the copied files and OBS launched. I downloaded the installer and chose the obs folder, after the installation, OBS still won't open.

**Comment by royshil** [comment]
@arize99 are you able to send OBS logs or the crash log?
you will find them in `C:\Users\<user>\AppData\Roaming\obs-studio\logs`

**Reply by arize99** [reply]
[2024-10-01 16-13-04.txt](https://github.com/user-attachments/files/17212182/2024-10-01.16-13-04.txt)


16:13:04.994: CPU Name: Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
16:13:04.994: CPU Speed: 2400MHz
16:13:04.994: Physical Cores: 2, Logical Cores: 4
16:13:04.994: Physical Memory: 16280MB Total, 5260MB Free
16:13:04.994: Windows Version: 10.0 Build 22000 (revision: 2538; 64-bit)
16:13:04.994: Running as administrator: true
16:13:04.995: Aero is Enabled (Aero is always on for windows 8 and above)
16:13:04.995: Windows 10 Gaming Features:
16:13:04.995: 	Game DVR: On
16:13:04.998: Sec. Software Status:
16:13:05.000: 	Microsoft Defender Antivirus: disabled (AV)
16:13:05.000: 	Reason Cybersecurity: enabled (AV)
16:13:05.000: 	Microsoft Firewall: disabled (FW)
16:13:05.001: Current Date/Time: 2024-10-01, 16:13:05
16:13:05.001: Browser Hardware Acceleration: true
16:13:05.001: Portable mode: false
16:13:05.973: OBS 24.0.3 (64-bit, windows)
16:13:05.973: ---------------------------------
16:13:05.973: ---------------------------------
16:13:05.973: audio settings reset:
16:13:05.973: 	samples per sec: 44100
16:13:05.973: 	speakers:        2
16:13:05.976: ---------------------------------
16:13:05.976: Initializing D3D11...
16:13:05.976: Available Video Adapters: 
16:13:05.979: 	Adapter 0: Intel(R) HD Graphics 520
16:13:05.979: 	  Dedicated VRAM: 134217728
16:13:05.979: 	  Shared VRAM:    4240754688
16:13:05.979: 	  output 0: pos={0, 0}, size={1366, 768}, attached=true
16:13:05.983: Loading up D3D11 on adapter Intel(R) HD Graphics 520 (0)
16:13:06.025: D3D11 loaded successfully, feature level used: b000
16:13:06.627: ---------------------------------
16:13:06.627: video settings reset:
16:13:06.627: 	base resolution:   1366x768
16:13:06.627: 	output resolution: 1920x1080
16:13:06.627: 	downscale filter:  Bicubic
16:13:06.627: 	fps:               30/1
16:13:06.627: 	format:            NV12
16:13:06.627: 	YUV mode:          601/Partial
16:13:06.627: NV12 texture support not available
16:13:06.627: Audio monitoring device:
16:13:06.627: 	name: Default
16:13:06.627: 	id: default
16:13:06.628: ---------------------------------
16:13:06.631: [CoreAudio encoder]: CoreAudio AAC encoder not installed on the system or couldn't be loaded
16:13:06.633: Failed to load 'en-US' text for module: 'decklink-ouput-ui.dll'
16:13:06.682: [AMF] AMF Test failed due to one or more errors.
16:13:06.682: Failed to initialize module 'enc-amf.dll'
16:13:06.715: obs_register_source: Tried to register obs_source_info with size 400 which is more than libobs currently supports (288)

**Comment by arize99** [comment]
pushed down to 1.1.7, the background remover installed and OBS lanched but the Background Remove effect was not in the effects list.

**Comment by royshil** [comment]
OBS is too old
```
16:13:05.973: OBS 24.0.3 (64-bit, windows)
```
Please upgrade to v30.2+

**Comment by arize99** [comment]
I updated OBS to the newest version and everything worked fine. Additionally, the newest release of the Background remover was perfect.
